### ISSUE TITLE
Markdown linter very sensible

### ISSUE THREAD

**Issue by maximilianong** [eclipse-tractusx/eclipse-tractusx.github.io#720]
To promote some headings of segments you want to make the first line in bold and not use h2 or h3 because you do not want to show the segment in the right navigation.

The linter is not allowing this:
<img width="695" alt="Bild (2)" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/83761111/d15aecf6-e5a7-4423-b1c1-7ed8d1454e8b">

Can we modify the linter or do you have any other idea on how to solve this @SebastianBezold / @mhellmeier ?

**Comment by mhellmeier** [comment]
Yes, you can configure the Linter as you want! The relevant file in this case is the `.markdownlint.yaml` in the root directory (see [here](.markdownlint.yaml)).

In your example, the `MD036` is the current problem. By adding it to the file with `false`, it will disable the linting rule.

**Comment by FaGru3n** [comment]
Hi @maximilianong if you use more `#` to just make it bold it will also work

```mardown
## h2 Heading

### h3 smaller heading

#### subsection

##### subsubsection
```

## h2 Heading
### h3 smaller heading
#### subsection
##### subsubsection

**Comment by SebastianBezold** [comment]
Hi,

while it is true, that we can adjust rules, I think this specific one is a good one. 
Reason: Accessibility.
Headlines are usually a much better anchor for screen-readers than bold text for example.

So If the visual is not too significantly different, I would prefer to keep that rule as is.

@maximilianong could you maybe provide screenshot for bold and headline versions?

**Comment by maximilianong** [comment]
> Hi @maximilianong if you use more `#` to just make it bold it will also work https://www.markdownguide.org/basic-syntax/#headings
> 
> ```
> ## h2 Heading
> 
> ### h3 smaller heading
> 
> #### subsection
> 
> ##### subsubsection
> ```
> 
> ## h2 Heading
> ### h3 smaller heading
> #### subsection
> ##### subsubsection

There you will run into an other error :D

When you have an H2 before and go straight to subsection:


<img width="1040" alt="Bild (3)" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/83761111/87d62b57-9e87-4789-9f8f-9c5c8e7be0b3">

**Comment by maximilianong** [comment]
> Hi,
> 
> while it is true, that we can adjust rules, I think this specific one is a good one. Reason: Accessibility. Headlines are usually a much better anchor for screen-readers than bold text for example.
> 
> So If the visual is not too significantly different, I would prefer to keep that rule as is.
> 
> @maximilianong could you maybe provide screenshot for bold and headline versions?

With bold - its not visible in the right navigation what I like more:
<img width="1245" alt="Bildschirmfoto 2024-02-28 um 11 14 29" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/83761111/6e8efc25-05ef-4212-a21b-fedf5b0d1224">

With h3 - it will be visible in the right navigation even when it's not a real section. Its blowing up the right navigation without any better anchoring.
<img width="1510" alt="Bildschirmfoto 2024-02-28 um 11 14 36" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/83761111/329324c4-f406-4e28-8dd1-798679926c56">

**Comment by SebastianBezold** [comment]
Does this help? https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#toc_min_heading_level
We can configure the min and headline level, that should be displayed in the TOC.
Can be added to the frontmatter of a `.md` file.
So still I would prefer this option instead of changing semantics of the page structure

**Comment by SebastianBezold** [comment]
Hi @nhaenis,

not sure, if i do understand your comment correctly. The setting I linked and the `max` version of it is something we control. This can be added to the frontmatter section of the markdown file.
Frontmatter ist the section on top of the file divided by `---`.
So you can set this to whatever level you feel fit for your case. DId you try that already?

**Comment by nhaenis** [comment]
@SebastianBezold yes i tried the configuration for `toc_min_heading_level`: 1 and 2 which seems not to make any difference in the TOC:
<img width="279" alt="image" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/124871958/a59ff1e9-c308-401e-8f89-9997b4339c0f">
with `toc_min_heading_level`: 3 the TOC's only showing up the ### level:
<img width="364" alt="image" src="https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/124871958/a3818678-163a-4615-b34d-6e55050b996d">
what we want for this page is only the level 1 (##) in the TOC which seems not to be possible to configure with this parameter because the documentation you posted says the minimum value for `toc_min_heading_level` is 2.

For me, there are only 2 options we have to choose between:
- We accept an unnecessarily blown up TOC
- Allow normal bold (sub-)headings that should not appear in the TOC (which is usually done on purpose to keep the TOC lean, otherwise you choose ###)

**Comment by SebastianBezold** [comment]
I think with what you want to achieve, you need to set both `toc_min_heading_level: 2` and `toc_max_heading_level: 2`

Example: 

```md
---
title: "TOC example"
---

content

## Level Two headline

content

### Level Three headline

content

#### Level Four headline

content
```

results in 

![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/7570790/571c9a7a-d32a-47ad-aae9-82414b912cee)


```md
---
title: "TOC example"
toc_min_heading_level: 2
toc_max_heading_level: 2
---

content

## Level Two headline

content

### Level Three headline

content

#### Level Four headline

content
```

results in 

![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/7570790/6ac31ce5-3881-4ac4-8576-70852204b0b5)

**Comment by nhaenis** [comment]
@SebastianBezold yes you're right - setting both min and max to 2 solves that issue in that case 👍 we have another page where we wanted to have a bold subheading under a H2 (not appearing in the TOC on purpose) whereas other H3s should show up - any ideas on how to handle that?
In my opinion, the rule is still too strict and bold headlines are always used intentionally. Normally you always use H3, the headings are then also addressable.

**Comment by stephanbcbauer** [comment]
is this here already solved?
